parser/src/main/java/parser/publio/NextPageUrlConsumer.java

```diff
@@ -38,23 +37,23 @@ public NextPageUrlConsumer(BlockingQueue urlToNextPageQueue,PageLoader pageLoade
 
     @Override
     public void run() {
-        logger.debug("start thread");
+        log.debug("start thread");
 
         while (true) {
             try {
                 String url = urlToNextPageQueue.take();
-                logger.debug("from urlToNextPageQueue " + url);
+                log.debug("from urlToNextPageQueue " + url);
 
                 if ("DONE".equals(url)) {
                     break;
                 }
 
                 bookDetailsUrls.addAll(getListToBookDetails(url));
             } catch (InterruptedException e) {
-                logger.error(e);
+                log.error(e);
             }
         }
-        logger.debug("exit thread");
+        log.debug("exit thread");
```

Comment:
I have this information in console/file pattern layout. [%t] gives me this information
(%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n)
It is from log4j2.xml